apps/deploy-web/src/components/layout/SidebarRouteButton.tsx

```diff
@@ -13,36 +13,62 @@ type Props = {
   route: ISidebarRoute;
   isNavOpen?: boolean;
   className?: string;
+  useNextLinkTag?: boolean;
 };
 
-export const SidebarRouteButton: React.FunctionComponent<Props> = ({ route, className = "", isNavOpen = true }) => {
+export const SidebarRouteButton: React.FunctionComponent<Props> = ({ route, className = "", isNavOpen = true, useNextLinkTag = true }) => {
   const pathname = usePathname();
   const isSelected = route.url === UrlService.home() ? pathname === "/" : route.activeRoutes.some(x => pathname?.startsWith(x));
 
   return (
     <li className={className}>
-      <Link
-        target={route.target ?? "_self"}
-        rel={route.rel ? route.rel : ""}
-        href={route.url}
-        className={cn(
-          buttonVariants({ variant: isSelected ? "secondary" : "ghost", size: "sm" }),
-          "flex w-full items-center justify-start text-current hover:no-underline",
-          {
-            ["font-bold"]: isSelected,
-            ["min-w-[initial] px-4 py-1"]: isNavOpen,
-            ["w-[45px] min-w-0 p-2"]: !isNavOpen
-          }
-        )}
-        data-testid={route.testId}
-      >
-        {!!route.icon && (
-          <span className={cn("z-[100] min-w-0", { ["m-[initial]"]: isNavOpen, ["mx-auto"]: !isNavOpen })}>
-            {route.icon({ className: cn({ ["text-primary font-bold"]: isSelected, ["mx-auto"]: !isNavOpen }, "text-xs") })}
-          </span>
-        )}
-        {isNavOpen && <span className="mb-1 ml-4 mt-1 min-w-0 flex-auto whitespace-nowrap">{route.title}</span>}
-      </Link>
+      {useNextLinkTag ? (
+        <Link
```

Comment:
Probably better to use something like `React.createElement(useNextLinkTag ? "Link" : "a", {attribute: stuff}, <div>Inner content</div>)`